src/Websites/Websites/ChangeLog.md

```diff
@@ -19,7 +19,8 @@
 -->
 ## Upcoming Release
 * Increased timeout for Publish-AzWebApp command
-* Fixed Set-AzWebApp isue with `Set-AzWebApp` when piping in Get-AzWebApp object [#21820]
+* Fixed Set-AzWebApp issue with `Set-AzWebApp` when piping in Get-AzWebApp object [#21820]
+* Added support for the PremiumMV3 tier to `New-AzResourceGroup` [#21933]
```

Comment:
The cmdlet should be 'New-AzAppServicePlan' instead of 'New-AzResourceGroup`, right?